### ISSUE TITLE
No focus indicator

### ISSUE THREAD

**Issue by beardcoder** [neos/neos-ui#1593]
### Description

No highlighting of focus elements.

### Steps to Reproduce

1. Open the neos-ui "mypage.xy/news"
2. Press the tab key

#### Expected behavior

After press the tab key you see the default browser visual focus indicator.

![neos ui](https://cdn.pbrd.co/images/H5uwXa0.png)

#### Actual behavior

No visual focus indiucator.

### Affected Versions

UI: dev-master 1.0.4
